packages/types/src/create/registry.ts

```diff
@@ -193,6 +193,10 @@ export class TypeRegistry implements Registry {
     return this.#knownTypes;
   }
 
+  public get unknownTypes (): Map<string, boolean> {
+    return this.#unknownTypes;
```

Comment:
Think it will make more sense not as Map. It really is an internal representation.

```suggestion
  public get unknownTypes (): string[] {
    return [...this.#unknownTypes.keys()];
```

With the relevant change in the interface as well. The TypeRegistry is bound to change, so really don't want the internal structures to bleed through interface use since that yields breaking changes.